prov/usnic/src/usdf_eq.c

```diff
@@ -171,11 +171,14 @@ usdf_eq_write_event(struct usdf_eq *eq, uint32_t event,
 }
 
 static ssize_t usdf_eq_readerr(struct fid_eq *feq,
-		struct fi_eq_err_entry *entry, uint64_t flags)
+		struct fi_eq_err_entry *buffer, uint64_t flags)
```

Comment:
> What would you suggest for the name? I see the manpage refer to this as buffer and it make sense to me but I can change this.

So it does, just leave it as-is then, since that will make more sense if you've just looked at the library prototype.